devito/operator.py

```diff
@@ -456,7 +456,7 @@ def apply(self, **kwargs):
         self.cfunction(*list(arguments.values()))
 
         # Output summary of performance achieved
-        summary = self.profiler.summary(dim_sizes, self.dtype)
+        summary = self.profiler.summary(arguments, self.dtype)
```

Comment:
so can we drop ``dim_sizes`` ?

Reply:
I am working towards dropping it, yes. I think this might be controversial with @mloubout though. I believe `arguments`, the other return value has everything we need.

Reply:
What d you mean drop dim sizes?